share/pvss/pvss.go

```diff
@@ -84,21 +90,33 @@ func EncShares(suite Suite, H kyber.Point, X []kyber.Point, secret kyber.Scalar,
 }
 
 // VerifyEncShare checks that the encrypted share sX satisfies
-// log_{H}(sH) == log_{X}(sX) where sH is the public commitment computed by
+// log_{h}(sH) == log_{X}(sX) where sH is the public commitment computed by
 // evaluating the public commitment polynomial at the encrypted share's index i.
-func VerifyEncShare(suite Suite, H kyber.Point, X kyber.Point, sH kyber.Point, encShare *PubVerShare) error {
+func VerifyEncShare(
+	suite Suite,
+	H kyber.Point,
+	X kyber.Point,
+	sH kyber.Point,
+	encShare *PubVerShare,
+) error {
```

Comment:
Can we please return a Bool instead of an error? see video below

Reply:
I exported and wrapped the errors